packages/web3-modal/src/theme/types.ts

```diff
@@ -127,7 +127,43 @@ interface AccountModalTheme extends SharedModalTheme {
 }
 type ConnectionModal = SharedModalTheme
 type HidDeviceModal = SharedModalTheme
-type TransactionsModalTheme = SharedModalTheme
+export interface TransactionsModalTheme extends SharedModalTheme {
+  card: {
+    background?: Pick<BackgroundStyles, 'success' | 'error'>
+    /**
+     * Colour of the `a` tag
+     */
+    url: string
```

Comment:
I think this should be inside of the `text` object down below

Reply:
can move, but not sure it makes sense since the `text` below pertains to the `statusPill`, and the `url` is for the explorer link

Reply:
i think in general anchor tags should all be the same colour. can be more generic and renamed to `anchor` but `url` is fine to me

Reply:
updated the fields name [4627020](https://github.com/PAST3LLE/monorepo/pull/77/commits/4627020a463a616a66c7c27b03e35b1d998b73fa)